### DISCUSSION TITLE
projection information on dwg file

### DISCUSSION THREAD

**Discussion by sree9578** [LibreDWG/libredwg#400] [Q&A] [answered]
Hi @rurban 

will i am able to get projection information from this dwg file?

**Comment by rurban** [comment, marked as answer]
If the DWG has this GEODATA information, then yes. See https://github.com/LibreDWG/libredwg/blob/344175ea9dbdbbcab4f9acc2412b382f5325a58d/src/dwg.spec#L5286
for the fields in GEODATA.

However I don't have much more info on this object, and only have limited examples to check against

**Reply by sree9578** [reply]
thanks
